shared/src/main/kotlin/com/egm/stellio/shared/model/EntityEvent.kt

```diff
@@ -1,8 +1,10 @@
 package com.egm.stellio.shared.model
 
+import java.net.URI
+
 data class EntityEvent(
     val operationType: EventType,
-    val entityId: String,
+    val entityId: URI,
```

Comment:
did you check events are correctly parsed when received from Kafka?

Reply:
yes